src/js/components/FormField/FormField.js

```diff
@@ -242,8 +242,12 @@ const FormField = forwardRef(
       formFieldTheme.disabled.border.color
     ) {
       borderColor = formFieldTheme.disabled.border.color;
-    } else if (error && themeBorder && themeBorder.error.color) {
-      borderColor = themeBorder.error.color || 'status-critical';
+    } else if (
+      error &&
+      formFieldTheme.error.border &&
+      formFieldTheme.error.border.color
```

Comment:
For backward compatibility reasons, we should still support `themeBorder.error.color` if provided, and ignore it if `formFieldTheme.error.border.color` is defined.

Reply:
Ok, I will update that tomorrow.